### ISSUE TITLE
Masked Language Modelling with a pretrained language model

### ISSUE THREAD

**Issue by spark157** [songlab-cal/tape#34]
Hi,

I'm interested to continue training an already pre-trained model using the masked language task. 

When I pass in the --from_pretrained flag it gives an error. I can see in `create_train_parser` there is no `add_argument` for the argument --from_pretrained. Would it be as simple as adding the functionality to read in the argument for --from_pretrained in main.py and then the pre-trained model will subsequently be loaded properly as the starting point for more training?

Thanks.

Scott

**Comment by rmrao** [comment]
Hi Scott,

The parsers are built somewhat hierarchically from a base parser. So the construction would be:

```python
base_parser = create_base_parser()
train_parser = create_train_parser(base_parser)
```

The base parser has the --from_pretrained flag. You could also add the flag manually, if you would prefer.
